src/Sushi.MailTemplate.SendGrid/QueuePersister.cs

```diff
@@ -16,53 +11,10 @@ public QueuePersister(string connectionString)
 
         public string ConnectionString { get; protected set; }
 
-        protected static string QueueCachePrefix = Guid.NewGuid().ToString();
-        public CloudQueue GetQueue(string name)
-        {
-            CloudStorageAccount storageAccount = CloudStorageAccount.Parse(ConnectionString);
-
-            // Create the queue client
-            CloudQueueClient queueClient = storageAccount.CreateCloudQueueClient();
-
-            // Retrieve a reference to a queue
-            CloudQueue queue = queueClient.GetQueueReference(name);
-
-            //did we already call createIfNotExists for this container?
-            string cacheKey = QueueCachePrefix + storageAccount.Credentials.AccountName + name;
-
-            var cache = System.Runtime.Caching.MemoryCache.Default;
-            if (!cache.Contains(cacheKey))
-            {
-                queue.CreateIfNotExists();
```

Comment:
CreateIfNotExists is removed, is this correct?

Reply:
re-added this code, was not supposed to be deleted